README.md

```diff
@@ -49,12 +49,21 @@ In order to start contributing, simply follow these 10 steps:
 2. Fork the [ACS UPB Mobile](https://github.com/acs-upb-mobile/acs-upb-mobile) repository.
 3. Clone the fork onto your system and open it as an Android Studio project.
 4. Don't forget to run `flutter pub get` to update the dependencies.
-5. Add yourself to the list of contributors in the `README.md` file. The recommended format is: `[FirstName LastName](link_to_github_profile)`.
-6. Commit and push the changes to the forked repository (*Ctrl*+*K* for committing, *Ctrl*+*Shift*+*K* for pushing in Android Studio).
-7. Create a pull request either from GitHub (you will see a message prompting you to create a PR when you open the forked repository) or Android Studio (VCS > Git > Create Pull Request).
-8. Wait for the PR to be approved.
-9. ???
-10. Profit!
+5. Create a new branch
+6. Add yourself to the list of contributors in the `README.md` file. The recommended format is: `[FirstName LastName](link_to_github_profile)`.
+7. Commit and push the changes to the forked repository (*Ctrl*+*K* for committing, *Ctrl*+*Shift*+*K* for pushing in Android Studio).
+8. Create a pull request either from GitHub (you will see a message prompting you to create a PR when you open the forked repository) or Android Studio (VCS > Git > Create Pull Request). Make sure the PR points from the branch you created to the master branch in the acs-upb-mobile repo.
+9. Wait for the PR to be approved.
+10. ???
```

Comment:
You can remove these steps and change the 'follow these 10 steps' part above accordingly. The joke doesn't make much sense anymore.